components/channel_header/channel_header.jsx

```diff
@@ -783,10 +793,7 @@ export default class ChannelHeader extends React.Component {
                     >
                         {dmHeaderIconStatus}
                         {dmHeaderTextStatus}
-                        <span
-                            onClick={Utils.handleFormattedTextClick}
-                            dangerouslySetInnerHTML={{__html: TextFormatting.formatText(channel.header, textFormattingOptions)}}
-                        />
+                        <span>{messageHtmlToComponent(formattedText, false, {mentions: false})}</span>
```

Comment:
I think here the handleFormattedTextClick is still needed. For example, I can't click a hashtag link.

Reply:
Oh yeah, that was an accidental removal. I'll add it back